Sources/ContentScopeScripts/dist/contentScope.js

```diff
@@ -3538,12 +3538,32 @@
   /*********************************************************
    *  Messaging to surrogates & extension
    *********************************************************/
+
+  /**
+   * @typedef enableSocialTrackerRequest
+   * @property {string} entity
+   *   The entity to unblock requests for (e.g. "Facebook").
+   * @property {bool} [isLogin=false]
+   *   True if we should "allow social login", defaults to false.
+   * @property {string} action
```

Comment:
Note: It's also a bug that `action` is not being passed, but that is only for Chrome MV3. I have follow-up changes MV3 changes anyway, so will address that with those.